src/javascript/node/sessionless.js

```diff
@@ -7,6 +7,26 @@ let getKeysFromDisk;
 
 const AsyncFunction = (async () => {}).constructor;
 
+const decimalToHex = (str) => {
```

Comment:
seems like this converts a number (string input) into hex

but it could be done with `toString(16)` for simplicity unless you want to do something specific with it

```js
const decimalToHex = (str) => {
    const decimal = parseInt(str);
    if (isNaN(decimal)) return '';
    return decimal.toString(16);
};
```

Reply:
you need this to convert JS BigInts to hex strings. The naming here could probably be cleaned up a bit though to have it be clearer what's going on.